### DISCUSSION TITLE
Make support for scavenging super easy

### DISCUSSION THREAD

**Discussion by bageldotjpg** [vectorwing/FarmersDelight#745] [Ideas]
It would be cool if the scavenging loot modifiers were updated to use entity type tags instead of a strict entity type to make them more easily expandable. Example:
The `farmersdelight:scavenging_feather` loot modifier could use a `farmersdelight:scavenging/feather` or `farmersdelight:scavenging_feather` or `farmersdelight:drops_feather` or `farmersdelight:scavenging/drops_feather` entity type tag, just something along those lines, instead of checking `minecraft:chicken` so other mods can just add things to those entity tags. Same could be applied to the other scavenging loot modifiers, especially like leather and string, but all of them would be useful.

**Comment by thelegitdolt** [comment]
omg bagel i'm your biggest fan

**Comment by vectorwing** [comment]
I had to double-check if the entity type predicate allowed for tag usage, and seems like it does.
Given that mods might indeed have extra mobs who might drop common items, this could be a good way to allow expansions. Though this might have to wait on 1.20, as entity loot modifiers are borked atm.
